apps/server/src/modules/board/service/board-do.service.ts

```diff
@@ -15,6 +16,8 @@ export class BoardDoService {
 		}
 
 		await this.boardDoRepo.delete(domainObject);
+
+		await domainObject.acceptAsync(this.deleteHookService);
```

Comment:
this only calls the hook on the initially deleted node, not on the descendants.

We either have to recursively go through all nodes in this function, or call this from the repository.

Reply:
Well, it should be called recursively. There's a test that describes that:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/e62674159b392ae8e611b5970908d235af9dddaf/apps/server/src/modules/board/service/delete-hook.service.spec.ts#L23

Reply:
Implementation here:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/e62674159b392ae8e611b5970908d235af9dddaf/apps/server/src/modules/board/service/delete-hook.service.ts#L22-L24